Makefile

```diff
@@ -103,26 +109,31 @@ sqlite:
 	@$(MAKE) migrate
 
 run:
-	${ACTIVATE} ./manage.py runserver
+	${MANAGE} runserver 8001 --noreload --insecure
```

Comment:
I just want this aligned with scancode.io. 8000 is the port used by the docker gunicorn. So it could not be used by a plain run. `--noreload` avoids several quirk when doing auto-reloads